### ISSUE TITLE
build: integrate moment-locales webpack plugin

### ISSUE THREAD

**Issue by dsebastien** [NationalBankBelgium/stark#85]
Integrate https://www.npmjs.com/package/moment-locales-webpack-plugin

**Comment by dsebastien** [comment]
Comment @christophercr:
We already remove all the locales we don't need from moment by using the ContextReplacementPlugin from webpack itself in this line. Basically this:
```
const ContextReplacementPlugin = require("webpack/lib/ContextReplacementPlugin");

// then in the webpack.commong.js we include it in the plugins array
new ContextReplacementPlugin(/moment[\/\\]locale$/, /(de|fr|en-gb|nl|nl-be)\.js/)
```

I've checked the source code of the moment-locales-webpack-plugin and in fact, they are doing exactly the same in this line. The good thing about this plugin is that it has some checks when the locales provided don't exist in Moment,
So the question is: what do we prefer? A new dependency in our build to have this plugin and make it easier to define which lolcales to remove? Or use a webpack built-in plugin to do that ourselves in a more verbose way?

**Comment by christophercr** [comment]
Blocked by #883

**Comment by SuperITMan** [comment]
Closed by #2563
